src/Trace/TraceSpan.php

```diff
@@ -46,12 +46,10 @@ class TraceSpan
      *      @type int $spanId The ID of the span. If not provided,
      *            one will be generated automatically for you.
      *      @type string $name The name of the span.
-     *      @type \DateTimeInterface|int|float|string $startTime Start time of the span in nanoseconds.
-     *            If provided as a string, it must be in "Zulu" format. If provided as an int or float, it is
-     *            expected to be a Unix timestamp.
-     *      @type \DateTimeInterface|int|float|string $endTime End time of the span in nanoseconds.
-     *            If provided as a string, it must be in "Zulu" format. If provided as an int or float, it is
-     *            expected to be a Unix timestamp.
+     *      @type \DateTimeInterface|int|float $startTime Start time of the span in nanoseconds.
```

Comment:
The string option was removed awhile ago because no one will be starting spans with a timestamp string or loading an instance of this class from an API call.